src/01_bignum.py

```diff
@@ -1,4 +1,9 @@
 # Print out 2 to the 65536 power
 # (try doing the same thing in the JS console and see what it outputs)
 
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+
+num = 2 ** 65536
```

Comment:
This was all you needed! 
Although, you didn't have to store them in a variable. 
print(2**65536)